src/firefox/preferences.js

```diff
@@ -65,13 +69,13 @@ prefs.common = {
 };
 
 // Prefs specific to Firefox for Android.
-prefs.fennec = {
+const prefsFennec:FirefoxPreferences = {
```

Comment:
I'm not [even remotely] a Flow expert, but we may be able to use [**eslint-plugin-flowtype**](https://www.npmjs.com/package/eslint-plugin-flowtype) to enforce this. Specifically the [`flowtype/space-after-type-colon`](https://www.npmjs.com/package/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-space-after-type-colon) rule.